hibike/mac_setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
```

Comment:
small nit: Maybe add some documentation about what's required to run this setup file (homebrew?)